src/utils/validations.ts

```diff
@@ -94,15 +94,24 @@ export function validateDomainOptions(options: any, validations: Array<DomainOpt
   }
 }
 
-export function validateRegexEntityParams(locale: string|undefined, pattern: string|undefined) {
+export function validateRegexEntityParams(
+  locale: string|undefined,
```

Comment:
Is `locale?: string` equivalent to this typedef?

Reply:
locale? means one do not need to pass the param but in this case here, it has be passed explicitly with `undefined`. That's because I went for positions params instead of a single object param, in which case, locale? would be it.